### ISSUE TITLE
finetuning on spherical vid - inconsistent tensor sizes

### ISSUE THREAD

**Issue by itsKaspar** [manuelruder/fast-artistic-videos#24]
Hi Manuel,
I'm running into a recurrent issue while finetuning for spherical videos
after a bit of training, during the run on validation set
I get an inconsistent tensor sizes error.
Here is the traceback : 

`Iteration 94000 / 120000, loss = 575304.800131	
Running on validation set ... 	
/home/kaspar/torch/install/bin/luajit: inconsistent tensor sizes at /tmp/luarocks_cutorch-scm-1-6847/cutorch/lib/THC/generic/THCTensorMath.cu:157
stack traceback:
	[C]: at 0x7f6350def6e0
	[C]: in function 'cat'
	train_video.lua:466: in function 'main'
	train_video.lua:557: in main chunk
	[C]: in function 'dofile'
	...spar/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00405d50`

and here is my command : 

`th train_video.lua \
-resume_from_checkpoint ~/fast-artistic-videos/models/monark.t7 \
-h5_file /media/kaspar/neural/coco2014/image_dataset.h5 \
-h5_file_video /media/kaspar/neural/video_dataset.h5 \
-image_model self \
-style_image $1 \
-loss_network /media/kaspar/neural/loss/vgg16.t7 \
-data_mix video:3,shift:1,zoom_out:1,single_image:5,vr:10 \
-num_frame_steps 0:1,50000:2,60000:4 \
-num_iterations 120000 \
-pixel_loss_weight 100 \
-style_image_size 900 \
-arch c9s1-32,d64,d128,R128,R128,R128,R128,R128,U2,c3s1-64,U2,c9s1-3 \
-gpu 0 \
-backend cuda \
-use_cudnn 1 \
-checkpoint_every 2000 \
-checkpoint_name ~/fast-artistic-videos/models/monark_vr`

I think I followed everything right, but still can't quite grasp the issue
thanks for your help :)))

**Comment by manuelruder** [comment]
It was a bug that occurred with the "image_model self" option. I fixed something similar for train already not forgot to check the validation code.
